core/tracepoint/new_spec.rb

```diff
@@ -0,0 +1,55 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+
+module ClassSpecs
+  class A
+    def foo; end
+  end
+end
+
+def test; 'test' end
+
+ruby_version_is '2.0' do
+  describe 'TracePoint.new' do
```

Comment:
We are still missing a spec for the passing an argument that's not string-like or symbol-like.

Reply:
okay, I have added a spec & updated the PR.